ui/apps/platform/cypress/integration/clusters/clusters.test.js

```diff
@@ -48,6 +49,24 @@ describe('Clusters page', () => {
             });
         });
     });
+
+    describe('when no secured clusters are added yet (only applies to Cloud Service)', () => {
+        it('should should redirect to the Clusters page', () => {
+            visitDashboardWithNoClusters();
+
+            cy.url().should('contain', `${clustersUrl}`);
```

Comment:
Please rewrite assertion according to majority pattern (118 versus 3 occurrences) and move into helper function because semantically related to heading assertion there, which is repeated here.

Also removes need to import url here. By the way, it can be exact match, true?

```js
cy.location('pathname').should('eq', clustersUrl);
```

Reply:
I intentionally took all assertions out of the helper function, because this is a one-off test, where the helper function is mainly just to abstract the API mocks and waiting. 
(I almost moved everything into test, instead of keeping the mocking in a helper function, but in the end made the subjective call that the setup distracted from the assertions, so that's where I drew the line in this special case.